android/AndroidManifest.xml

```diff
@@ -50,5 +50,5 @@
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
     <uses-feature android:glEsVersion="0x00020000"/>
-    <uses-sdk android:minSdkVersion="9"/>
+    <uses-sdk android:minSdkVersion="26"/>
```

Comment:
We've been using 26 for other projects lately, so it was reflex.  23 is probably a sensible lower level, continues Marshmallow support.  Anything before that is a pretty rare device.  I'll test out some lower levels locally.

`c++_static` is more tied to the NDK level.  `stlport_static` has been deprecated (and removed as of r18).

Reply:
It appears android-9 does work, although pops up a variety of warnings on modern devices.  I won't push on this if you want to keep it low.

Reply:
It would be nice to not have warnings on newer devices. @afrantzis does 23 sounds reasonable? Otherwise @cnorthrop can you check if there are no warnings on 19 or 21? 19 accounts for >95% of users and 21 about 90%. See https://developer.android.com/about/dashboards

Will let Cody handle the fix here.

Reply:
`android-23` makes a lot of sense.  That's when runtime permissions were added, which allows the app to launch and start cleanly.  Anything before that presents a full screen permission request when you launch the app.  I switched to that in the [branch](https://github.com/cnorthrop/glmark2/tree/android_updates) that can follow this PR.